client/starwhale/base/bundle.py

```diff
@@ -76,9 +76,29 @@ def list(
         project_uri: URI,
         page: int = DEFAULT_PAGE_IDX,
         size: int = DEFAULT_PAGE_SIZE,
+        filters: t.Union[t.Dict[str, t.Any], t.List[str]] = {},
     ) -> t.Tuple[t.Dict[str, t.Any], t.Dict[str, t.Any]]:
         _cls = cls._get_cls(project_uri)
-        return _cls.list(project_uri, page, size)  # type: ignore
+        _filter = cls.get_filter_dict(filters, cls.get_filter_fields())
+        return _cls.list(project_uri, page, size, _filter)  # type: ignore
+
+    @classmethod
+    def get_filter_dict(
+        cls, filters: t.Union[t.Dict[str, t.Any], t.List[str]], fields: t.List[str] = []
+    ) -> t.Dict[str, t.Any]:
+        if isinstance(filters, t.Dict):
+            return {k: v for k, v in filters.items() if k in fields}
+
+        _filter_dict = {}
+        for _f in filters:
+            _item = _f.split("=", 1)
+            if _item[0] in fields:
+                _filter_dict[_item[0]] = _item[1] if len(_item) > 1 else True
```

Comment:
Does `True` as the default value meet the unexpected scenario?

For example:
`--filter name` will be converted into `--filter name=True`, `True` is a bool value which is inappropriate for `name` value.

Reply:
done